.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules/
 *.log
 .env
+assets/images
```

Comment:
You don't want to gitignore these! These are your static image assets, things like logos / icons etc.